app/client/src/views/HomeView.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <div class='home'>
     <h1>Welcome to beebop!</h1>
-    <div v-if='user' class='auth'>
-      <LoginPrompt v-if='!loggedIn' />
-      <GreetingAndLogout v-if='loggedIn' />
+    <div class='auth'>
```

Comment:
I'm a bit unclear about how the changes in this view relate to the apiService?

Reply:
Previously I had commited the response object (with success, data, errors) in total to the user state. I first checked with the computed property `loggedIn`, whether the status was 'success' and the user was actually logged in. Since that check is now done by the api class, I do only have a user saved in state when this was a successful request. So `loggedIn` became redundant, since the user state means the same now.